nautobot/ui/src/utils/string.js

```diff
@@ -1,6 +1,12 @@
 export function toTitleCase(str, separator = " ") {
     return str
         .split(separator)
-        .map((x) => (x ? x[0].toUpperCase() + x.slice(1) : ""))
+        .map((x) =>
+            x
+                ? x === "Ip"
```

Comment:
Is that the best approach we can do? We'll have to solve the same problem for `vlan`, `vrf`, `vm interface`, etc. Would it be better to do something through the detail_view_config or list_display_fields in some way?

Reply:
Let me know if this is an acceptable way of doing this. https://github.com/nautobot/nautobot/pull/4520/commits/aafb031fc12ded8c1c80863ae51c09e7ec14aa06#diff-7eb4de91128d426a0c3b51d50705d7ba4b1033169ec88617031657896728d8e1R73-R75.
Basically by using the "label" attribute on the serializer field for display in the UI.

Reply:
Yes! definitely the direction we should head.